docs/man/git-lfs-config.adoc

```diff
@@ -145,6 +145,16 @@ to simple HTTP.
 If set to true, this enables resumable uploads of LFS objects through
 the tus.io API. Once this feature is finalized, this setting will be
 removed, and tus.io uploads will be available for all clients.
+* `lfs.transfer.batchSize`
++
+The number of objects to download/upload sent in a single batch request
+to the LFS server. Default is 100.
++
+This value should be changed with caution, as it can have a significant
+impact on the performance of the LFS server and the server is free to
+return an HTTP 413 status code if this value is too high as the Batch
+API specification states.
+
```

Comment:
This is great documentation, thank you!

I realized on a second read-through that I should perhaps ask if you wouldn't mind shifting this down a bit so that it's organized with the other `lfs.transfer.*` settings, like `lfs.transfer.maxRetries` and the ones following that.  Our listing of configuration options is, sadly, not sorted alphabetically within each section, but if we at least keep the most closely-related entries together, I think that will help people a bit.